components/Header/index.jsx

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+import { useState } from 'react';
+
+import Sidebar from '@/components/Sidebar';
+import logo from '@/public/logo-small.svg';
+
+import useUser from 'hooks/useUser';
+import useProfile from 'hooks/useProfile';
+
+export const headerTestIds = {
+  base: 'header',
+  logo: 'header__logo',
+  userAvatar: 'header__user-avatar',
+};
+
+const Header = () => {
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+  const { user } = useUser({});
+  const profile = useProfile(user);
+
+  if (!profile) {
+    return null;
+  }
+
+  return (
+    <div className="flex justify-between mb-[42px]" data-test-id={headerTestIds.base}>
+      <div className="h-8 ml-8 mt-[34px]" data-test-id={headerTestIds.logo}>
+        <Image src={logo} alt="logo" />
+      </div>
+      <div
+        className="w-9 h-9 mt-8 mr-8 z-[1] cursor-pointer"
+        onClick={() => setIsSidebarOpen(value => !value)}
+        data-test-id={headerTestIds.userAvatar}
+      >
+        <Image src={profile.avatarUrl} alt="profile avatar" layout="fixed" width={36} height={36} />
```

Comment:
Should this avatar round?

```suggestion
        <Image className="rounded-full" src={profile.avatarUrl} alt="profile avatar" layout="fixed" width={36} height={36} />
```

The current style | The design
--- | ---
![Screen Shot 2565-04-11 at 15 14 08](https://user-images.githubusercontent.com/1772999/162693858-f70e3db5-2d7e-4517-b32b-9029027bb76f.png) |  ![Screen Shot 2565-04-11 at 15 14 20](https://user-images.githubusercontent.com/1772999/162693875-f428d7e2-c55a-4c62-9000-936856795281.png)

Reply:
I missed it completely, maybe because of the fact that the logo I have been testing so far is already rounded 😝 
done in 2125544666199978509632db13f61e3cd5903fa6